docs/review-process.md

```diff
@@ -87,6 +87,20 @@ the end-user getting your improvement faster.
    clear, and concise title, and add an extensive description of your change.
    Be sure to add a motivation (or use case) to your PR, so the reviewer can
    understand why you are making this change (or why you make certain decisions).
+   
+7. **Update dependency in a standalone PR.**
+   When you need to bump a dependency, try to do so in a standalone PR. Only
+   compatilibity code adjustments or related bug fixes should be included in the
```

Comment:
```suggestion
   compatilibity code adjustments or small related bug fixes should be included in the
```
A bugfix can be either small or gigantic, so if possible I would only thing small ones are nice to have combined